api/web3server.go

```diff
@@ -924,24 +925,32 @@ func (svr *web3Handler) traceTransaction(ctx context.Context, in *gjson.Result)
 		disableStorage = options.Get("disableStorage").Bool()
 		enableReturnData = options.Get("enableReturnData").Bool()
 	}
-	cfg := &logger.Config{
-		EnableMemory:     enableMemory,
-		DisableStack:     disableStack,
-		DisableStorage:   disableStorage,
-		EnableReturnData: enableReturnData,
+	cfg := &tracers.TraceConfig{
```

Comment:
Why not set `Tracer` and `Timeout`, compared with `tracCall`?

Reply:
If users have this requirement, they can add.

Reply:
I mean, does it support tracer and timeout config in traceTransaction just like what we supporting in traceCall?